components/brave_rewards/resources/rewards_panel/components/icons/refresh_status_icon.tsx

```diff
@@ -0,0 +1,13 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+export function RefreshStatusIcon () {
+  return (
+    <svg className='icon' fill='currentColor' viewBox="0 0 14 16" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I've always Reactified the SVG (convert double-quotes to single and change kebab-case property names to snakeCase), but apparently the linter doesn't mind - so I don't suppose it's important.

Reply:
Thanks, updated. Might as well keep it consistent.